Dockerfile

```diff
@@ -1,3 +1,4 @@
+#TODO: Is there a reason we're using 3.11 vs 3.12/3.13
```

Comment:
@TibsAtWork  - Is there a reason that we're using 3.11 versus more recent versions?